### PR TITLE
Support specify reference build from different job - UI

### PR DESCRIPTION
**Pull request by awang-parasoft**

### CODE REVIEW

**Comment by lli-parasoft** [comment]
I'm reviewing this

**Comment by lli-parasoft** [comment]
> I'm reviewing this

Ready to merge

File: src/main/webapp/help/parasoftCoverage/help-referenceJob.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div>
+    Specify a reference job to use if "Quality gates" is configured. To use the current job, leave this field
```
Comment (yshi-parasoft) [inline src/main/webapp/help/parasoftCoverage/help-referenceJob.html:2]: Change the message after "-" as:
which includes the names of all items in the hierarchy from the root Jenkins instance to the job, separated by '/'."

Comment (awang-parasoft) [reply src/main/webapp/help/parasoftCoverage/help-referenceJob.html:2]: The message is updated.

There's also some docs in Jenkins User Handbook.

[Referencing another project by name](https://www.jenkins.io/doc/book/using/referencing-another-project-by-name/#referencing-components-inside-projects)

Comment (yshi-parasoft) [reply src/main/webapp/help/parasoftCoverage/help-referenceJob.html:2]: According to what Alfie said, supporting the "relative paths" feature mentioned in the link requires extra work, but at the moment, we don't know the solution. I believe we don't need to align with all the methods supported by Jenkins. Users using the full name should be sufficient unless there are user complaints.
@rwang-parasoft Please merge this pull request if you don't have any other concerns.

**Pull request merged**